contributing/DOCS-CONTRIBUTING.md

```diff
@@ -68,45 +71,203 @@ documentation efforts, who are happy to help you get started.
+    an active maintainer, they will be moved into the "community samples"
+    folder.
+
+  * **Community owned and maintained samples**: Code samples that have
+  been contributed by Knative community members. These samples might not receive
+  regular maintenance. It is possible that a sample is no longer current and
+  is not actively maintained by its original author. While we encourage a
+  contributor to maintain their content, we acknowledge that it's not always
+  possible for certain reasons, for example other commitments and time
+  constraints.
+
+   While a sample might be out of date, it could still provide assistance and
+   help you get up-and-running with certain use-cases. If you find that
+   something is not right, contains out-dated info, please help us become aware
+   of the issue. In which case that sample might be fix if bandwidth or resource
+   exists, or taken down and archived into a past release branch (where it
+   hopefully worked last).
+
+#### Choosing the correct branch
+
+  It is likely that your docs contribution is either for new or changed
+  features in the product, or for a fix or update existing content.
+
+  * **New or changed features**:
+    If you are adding or updating documentation for a new or changed feature,
+    you likely want to open your PR against the `master` branch. All pre-release
+    content for active Knative development belongs in
+    [`master`](https://github.com/knative/docs/tree/master/).
+
+  * **Fixes and updates**:
+    If you find an issue in a past release, for example a typo or out-of-date
+    content, you likely need to open multiple and subsequent PRs. If not a
+    followup PR, at least add the "`cherrypick` labels" to your original PR
+    to indicate in which of the past release that your change affects.
+
+     For example, if you find a typo in a page of the `v0.5` release, then that
+     page in the `master` branch likely also has that typo.
+
+     To fix the typo:
+     1. You open a PR against
+        [`master`](https://github.com/knative/docs/tree/master/).
+     1. Add one or more `cherrypick-#.#` labels to that PR to indicate which
+        of the past release branches should also be fixed.
+     1. If you want to complete the fix yourself (best practice), you then open
+        a subsequent PR by running `git cherry-pick [OriginalPRCommit#]` against
+        the [release-0.5](https://github.com/knative/docs/tree/release-0.5).
+        Optionally (and depending on workload and available bandwidth), one of
+        the Knative team members might be able to help handle the
+        `git cherry-pick` in order to push that same fix into a affected release
+        branch(es).
+
+  For a list of the available branches in the `knative/docs` repo, see
+  [Documentation Releases](https://github.com/knative/docs/blob/master/doc-releases.md).
+
+## Assigning reviewers
+
+For both documentation and code samples, you should assign your PR to a single
```

Comment:
Right now, Prow automatically assigns two reviewers from the list of docs approvers and reviewers: https://github.com/knative/docs/blob/master/OWNERS_ALIASES

Reply:
Prow is nice and mostly reliable but the goal here is to address the lack of SME eyes on certain items, particularly code sample related items. Instead of maintainers manually assigning items (especially when they are from peers) it would expedite a review if the author assigned a reviewer immediately.

Reply:
Oh, my point was that you said explicitly to only assign it to one assign, and I was just pointing out that Prow automatically assigns two, so a person could see the auto-assignments as be like, "Oops, I need it to only be assigned to one person."

But also there's some restrictions on who is able to assign issues/PRs -- I *think* you have to be a member of the Knative org, so some contributors (esp. first timers) might not be able to assign anyone themselves.

Reply:
ah, good point. others can use `/assign` flag with prow, will add that

Reply:
done ptal